docs/querying/using-caching.md

```diff
@@ -0,0 +1,90 @@
+---
+id: using-caching
+title: "Using query caching"
```

Comment:
just a thought: this page is now entirely config instructions, but it appears under a "Concepts" heading in the left TOC. Not sure how much of an issue that is -- there are other config instructions under that heading -- but it's more apparent now with this standalone page.. We might want to rethink that heading or page locations in the TOC, perhaps in another PR.

Reply:
let's do that in another PR, I think.